MHacks/RoundedView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RoundedView.swift
+//  MHacks
+//
+//  Created by Connor Krupp on 9/11/16.
+//  Copyright © 2016 MHacks. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class RoundedView: UIView {
```

Comment:
Yeah you can. But that's built on KVC (sketchy) and is a pretty obscure feature. I'd prefer it written in code for clarity and discoverabily.